components/clm/src/biogeophys/SoilStateType.F90

```diff
@@ -881,11 +881,13 @@ subroutine Restart(this, bounds, ncid, flag)
             dim1name='column', dim2name='levgrnd', switchdim=.true., &
             long_name='soil matric potential', units='mm', &
             interpinic_flag='interp', readvar=readvar, data=this%smp_l_col)
+       if(flag=='read' .and. .not.readvar) this%smp_l_col = -2.5e3_r8
 
        call restartvar(ncid=ncid, flag=flag, varname='HK', xtype=ncd_double,  &
             dim1name='column', dim2name='levgrnd', switchdim=.true., &
             long_name='hydraulic conductivity', units='mm/s', &
             interpinic_flag='interp', readvar=readvar, data=this%hk_l_col)
+       if(flag=='read' .and. .not.readvar) this%hk_l_col = 1.e-9_r8
```

Comment:
Same comment as above.  What is the number and why are you assigning it to that variable here?

Reply:
Added comments. Thanks!